Examples/JamitFoundationExample/App/Sources/ActionView/ActionViewController.swift

```diff
@@ -0,0 +1,48 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import JamitFoundation
+import UIKit
+
+final class ActionViewController: StatefulViewController<ActionViewControllerViewModel> {
+    @IBOutlet weak var actionViewContainer: UIView!
+
+    private lazy var actionView: ActionView<ImageView> = {
+        let actionView = ActionView<ImageView>.instantiate()
+        return actionView
+    }()
```

Comment:
If you do configure nothing in the closure, you can reduce this to single line. No closure is needed.

Reply:
Done.